docs/DeveloperGuide.md

```diff
@@ -1,38 +1,135 @@
-# Developer Guide
+# Developer Guide for SpendSwift
 
 ## Acknowledgements
-
-{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
-
-## Design & implementation
-
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
-
-
-## Product scope
-### Target user profile
-
-{Describe the target user profile}
-
-### Value proposition
-
-{Describe the value proposition: what problem does it solve?}
+{List here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well.}
+
+## Design & Implementation
+### Sequence Diagrams
+#### add-expense
```

Comment:
![image](https://github.com/user-attachments/assets/4b529e92-d33e-4a67-8aff-e4d9e36b11f2)
Calling them themselves here is missing the activation bar.